queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '#2017-05-12#' AND '#2018-11-15#';
```

Comment:
- [ ] Kindly notice that the query should list the name of all animals born between 2016 and 2019.

```sql
SELECT name FROM animals WHERE EXTRACT(year FROM date_of_birth) BETWEEN 2016 AND 2019;
```